package.json

```diff
@@ -17,12 +17,12 @@
     "./lib/styles.css": "./lib/styles.css",
     ".": {
       "types": "./lib/index.d.ts",
-      "default": "./lib/bundle.js"
+      "default": "./lib/index.js"
     }
   },
-  "browser": "./lib/bundle.js",
-  "main": "./lib/bundle.js",
-  "module": "./lib/bundle.js",
+  "browser": "./lib/index.js",
+  "main": "./lib/index.js",
+  "module": "./lib/index.js",
```

Comment:
Do we need all three?

Reply:
Probably not at this point, it's just there for older bundlers, but we could try removing those.